wiki/Beatmap_Editor/Timing/en.md

```diff
@@ -1,41 +1,21 @@
-
-Again after the button is pressed enough, BPM sensitivity will change from 0,1 and 2.
+![Timing menu](img/Timing_base.jpg "Timing menu")
 
--   BPM Sensitivity 0: 999,00
--   BPM Sensitivity 1: 999,90
--   BPM Sensitivity 2: 999,99
-    -   *9 = random number / 0 = unaffected number.*
+The timing for a [beatmap](/wiki/Beatmaps) consists of the song's offset and BPM (beats per minute). The song's offset is the amount of time between the start of an audio file and the first downbeat in the song. The song's BPM is essentially how fast a song is.
 
-**The button «T» may also be used instead of «Tap Here!»**
+Correct timing is imperative in the beatmapping process to allow placing objects in time with the beat.
 
-Timing Setup
-------------
+## Setup
 
-![The Timing Setup dialog showing a map with a fluctuating BPM and sampleset changes.](img/TimingSetup.jpg "The Timing Setup dialog showing a map with a fluctuating BPM and sampleset changes.")
+![Screenshot of the "Timing and Control Points" window](img/TimingSetup.jpg "The timing setup dialog shows the beatmap's fluctuating BPM and sampleset changes")
 
-Timing sections allow you to vary the timing across the song. There are two types of timing sections, a regular timing section and an inherited timing section.
+Timing sections allow you to vary the timing across the song. There are two types of timing sections: a regular timing section and an inherited timing section.
 
-A regular timing section is displayed as a red line in the editor. Every map has at least one. Some maps change their BPM during the song, however, and require multiple regular timing sections as a result.
+A regular timing section is displayed as a red line in the editor. Every beatmap must have at least one. Some maps change their BPM during the song, however, and require multiple regular timing sections as a result.
 
-Inherited timing sections are displayed as a green line in the editor. These do not actually change the song's timing. What inherited timing sections do is let you change the map's [hitsound](/wiki/Glossary)'s volume, the hitsounds themselves, [slider speed](/wiki/Beatmap_Editor/Slider), etc. for that section without modifying the song's timing (which would happen with a regular timing section).
+Inherited timing sections are displayed as a green line in the editor. They do not allow changes to a beatmap's timing, but focus on adjusting the song's timing. What inherited timing sections do is let you change the beatmap's [hit sound](/wiki/hit_sound)'s volume, the hit sounds themselves, [slider speed](/wiki/Slider), etc. for that section without modifying the song's timing (which would happen with a regular timing section).
```

Comment:
- The slider link is correct when displayed live.
- The `hit_sound` link should go to the glossary, but it is correct when displayed live.